tests/unit/ssg_test_suite/test_matches_platform.py

```diff
@@ -80,16 +80,16 @@ def test_list_combined_match():
 
 
 def test_list_combined_match_2():
-    scenario_platforms = ["Debian 8", "multi_platform_ubuntu", "openSUSE",
+    scenario_platforms = ["multi_platform_ubuntu", "openSUSE",
```

Comment:
```suggestion
    scenario_platforms = ["Debian 8", "multi_platform_ubuntu", "openSUSE",
```

If the Debian platform is removed, then the CPE has no reason to match any of the remaining platforms.